VideoLocker/src/main/java/org/edx/mobile/util/AppConstants.java

```diff
@@ -9,6 +9,7 @@
     public static boolean videoListDeleteMode = false;
 
     public static final String VIDEOLIST_BACK_PRESSED = "offline_video_back_pressed";
+    public static final String NON_BREAKING_HYPHEN = "\u2011";
```

Comment:
If we're defining constants for Unicode characters, then this should be in a dedicated class. They should also be defined as `char` instead of `String`, regardless of whether you actually need a String representation in the implementation (it can be converted there).